### DISCUSSION TITLE
[PSD2] Exercise 1.52/1.53

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#126] [Esercizi (A-L)]
**Exercise 1.52** Perform the following additions of unsigned binary numbers. Indicate whether or not the sum overflows a 4-bit result.

(a) $1001_2 + 0100_2$
(b) $1101_2 + 1011_2$

**Exercise 1.53** Perform the following additions of unsigned binary numbers. Indicate whether or not the sum overflows a 8-bit result.

(c) $10011001_2 + 01000100_2$
(d) $11010010_2 + 10110110_2$

**Comment by Elia-Belli** [comment]
**Somma tra Unsigned**
(a) $1001_2 + 0100_2 = 1101_2$
(b) $1101_2 + 1011_2 = 11000_2$ (Overflow)
(c) $10011001_2 + 01000100_2 = 11011101_2$
(d) $11010010_2 + 10110110_2 = 110001000_2$ (Overflow)

**Comment by niccolo-fato** [comment]
![photo_2024-11-15_18-06-11](https://github.com/user-attachments/assets/f299cd0e-c661-4bd9-99dc-15ded5b5f11e)
